GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -94,9 +89,7 @@ protected GVRHybridObject(GVRContext gvrContext, long nativePointer,
         mGVRContext = gvrContext;
         mNativePointer = nativePointer;
 
-        synchronized (sReferenceSet) {
-            sReferenceSet.add(new GVRReference(this, nativePointer, cleanupHandlers));
-        }
+        gvrContext.registerHybridObject(this,nativePointer, cleanupHandlers);
```

Comment:
missing a space after "this,". It's a good time to fix it now if this patch is going to be updated.